website/docs/components/alert/partials/content/content.md

```diff
@@ -0,0 +1,151 @@
+## General recommendations
+
+- Keep the title short, as this will be the most prominent element when users scan the alert.
+- Avoid ending the title with a period.
+- Alert descriptions should be short but clear enough to explain what’s happening. We recommend keeping messages under 90 characters.
+- For warning and critical alerts, guide the users on how to prevent or fix the issue.
+
+## Actions
+
+Use small buttons to avoid competing with other actions on the page. Use more than two actions sparingly.
+
+We recommend using the `secondary` button variant for primary actions and the `tertiary` button variant for secondary actions.
+
+<Hds::Alert @type="inline" as |A|>
+  <A.Title>Recommended button usage</A.Title>
+  <A.Description>Lorem ipsum dolar sit amet.</A.Description>
+  <A.Button @text="Your action" @color="secondary" @onClick={{this.noop}} />
+  <A.Button @color="tertiary" @icon="arrow-right" @iconPosition="trailing" @text="Tertiary" />
+</Hds::Alert>
+
+For content guidelines on actions, refer to [Button](/components/button) and 
+[Link](/components/link/standalone) documentation.
+
+### Usage of critical buttons
+
+Avoid using critical buttons in alerts. We handle the prominence and importance via the styling of the alert container itself. If you need to confirm that the user intended to interact with the action, consider displaying a confirmation modal.
+
+!!! Dont
+<Hds::Alert @type="inline" @color="critical" as |A|>
+  <A.Title>Critical button usage</A.Title>
+  <A.Description>Lorem ipsum dolar sit amet.</A.Description>
+  <A.Button @text="Your action" @color="critical" @onClick={{this.noop}} />
+</Hds::Alert>
+!!!
+
+## Links
```

Comment:
Within the Alert components, "Actions" refer to both Buttons and Links within the "Footer" (although not defined as a footer) of the component. For this reason, I recommend nesting the "Links" section under the "Actions" section and keeping it related to the standalone link found in the "Footer." You can then move all new content about links within the description to a new section (H2) "Description." This may also be more scalable because it allows us to add more content about the description later. 

So essentially, the new content structure would look something like this: 

- Actions
  - Buttons
    - Usage of critical buttons
  - Links
    - Within page and inline Alerts
- Description

Reply:
Got it, makes sense